config/locales/candidate_interface/gcse.yml

```diff
@@ -9,6 +9,7 @@ en:
         gcse: GCSE
         gce_o_level: O level
         scottish_national_5: Scottish National 5
+        international_baccalaureate_middle_years_programme: International Baccalaureate Middle Years Programme
```

Comment:
```suggestion
        international_baccalaureate_middle_years_programme: International Baccalaureate Middle Years Programme (IB MYP)
```

Reply:
the acronym has been used on the following page so it's good practice to give the acronym here